dashboards-observability/public/components/visualizations/charts/logs_view/logs_view.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import './logs_view.scss';
+import React, { useContext } from 'react';
+import { EuiAccordion, EuiPanel, EuiSpacer, htmlIdGenerator } from '@elastic/eui';
+import { TabContext } from '../../../event_analytics/hooks';
+
+export const LogsView = ({ visualizations }: any) => {
+  const { explorerData } = useContext<any>(TabContext);
+  const { dataConfig = {} } = visualizations?.data?.userConfigs;
+  const isTimeEnabled =
+    dataConfig?.chartStyles?.time !== undefined ? dataConfig?.chartStyles?.time : true;
+  const isWrapLinesEnabled =
+    dataConfig?.chartStyles?.view !== undefined && dataConfig?.chartStyles?.view === 'wrapLines'
+      ? true
+      : false;
+  const isPrettifyJSONEnabled =
+    dataConfig?.chartStyles?.view !== undefined && dataConfig?.chartStyles?.view === 'prettifyJSON'
+      ? true
+      : false;
+  const isLogDetailsEnabled =
+    dataConfig?.chartStyles?.enableLogDetails !== undefined
+      ? dataConfig?.chartStyles?.enableLogDetails
+      : true;
+  const isOldestFirst = dataConfig?.chartStyles?.order === 'oldestFirst' ? true : false;
```

Comment:
check if this work ` const isOldestFirst = dataConfig?.chartStyles?.order === 'oldestFirst'`